open-metadata-resources/open-metadata-labs/ui-labs/ui-asset-search.ipynb

```diff
@@ -0,0 +1,290 @@
+    "\n",
+    "As part of the huge business transformation that Coco Pharmaceuticals has embarked on, they\n",
+    "have created a data lake for managing data for research, analytics, exchange between their internal organizations and business partners (such as hospitals).  As a result, the data lake has to be\n",
+    "designed to handle a wide variety of data, including some highly sensitive and regulated data.\n",
+    "\n",
+    "In this lab we use first data engine to import pre-defined assets. Afterwards, the UI where we can search for assets is shown. The main character engaged in this lab is the data scientist named Callie Quartile."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img style=\"margin-left: 7%;\" src=\"../images/callie-quartile.png\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> **Callie Quartile:** Data Scientist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up\n",
+    "\n",
+    "Coco Pharmaceuticals make widespread use of ODPi Egeria for tracking and managing their data and related assets.\n",
+    "Figure 1 below shows their metadata servers and the Open Metadata and Governance (OMAG) Server Platforms that are hosting them.  Each metadata server supports a department in the organization.  The servers are distributed across the platform to even out the workload.  Servers can be moved to a different platform if needed.\n",
+    "\n",
+    "![Figure 1](../images/coco-pharmaceuticals-systems-omag-server-platforms.png)\n",
+    "> **Figure 1:** Coco Pharmaceuticals' OMAG Server Platforms\n",
+    "\n",
+    "The code below sets up the network addresses for the three platforms.  This varies depending on whether you are running them locally, through **docker-compose** or on **kubernetes**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run ../common/environment-check.ipynb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "Callie is using the data lake operations metadata server called `cocoMDS1`. This server is hosted on the Data Lake OMAG Server Platform.\n",
+    "\n",
+    "Check that `cocoMDS1` is running.  \n",
+    "\n",
+    "If any of the platforms are not running, follow [this link to set up and run the platform](https://egeria.odpi.org/open-metadata-resources/open-metadata-labs/).  If any server is reporting that it is not configured then\n",
+    "run the steps in the **[Server Configuration](../egeria-server-config.ipynb)** lab to configure\n",
+    "the servers.  Then re-run the previous step to ensure all of the servers are started."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
+    "## Exercise 1\n",
+    "\n",
+    "### Importing IGC events\n",
+    "\n",
```

Comment:
I'm uncomfortable with only referring to IGC as the example. We need a generic version ... and I think it's then ok to also add IGC specific, but IGC only sends the wrong message IMO about being open - at least if this is in the 'core' project (as opposed to other sandpit/3rd party repos?)

Reply:
The primary reason why these mock events are being used is that they enable us to quickly demonstrate asset lineage in a follow-up tutorial, as we already have the right data for that. Do you think for the short term it is fine to leave the IGC part here, and add other sources in the future?